index.js

```diff
@@ -1,10 +1,4 @@
 var imagemin = require('imagemin');
-var imageminGifsicle = require('imagemin-gifsicle');
-var imageminMozjpeg = require('imagemin-mozjpeg');
-var imageminOptipng = require('imagemin-optipng');
-var imageminSvgo = require('imagemin-svgo');
-var imageminPngquant = require('imagemin-pngquant');
-var imageminWebp = require('imagemin-webp');
```

Comment:
And here.

Reply:
Strange, didn't see them before, maybe some caching of the browser.
Thanks